### PR TITLE
#168049987 Integrate HoundCI with badge attached to readme

### PR DESCRIPTION
**Pull request by amilykassim**
#### What does this PR do?
Configure hound-ci
#### Description of Task to be completed?
Setup ci
#### How should this be manually tested?
Check the badge on readme file and the .hound.yml
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168049987](https://www.pivotaltracker.com/story/show/168049987)
#### What are the relevant Github Issues?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions: N/A

### CODE REVIEW

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
+  ignore_file: .eslintignore
```
Comment (gik-hub) [inline .hound.yml]: You can add an empty line here for the sake of Standards.

Otherwise, everything else is quite in order.

**Review by mucyomiller — APPROVED** [review]
try to fix hound ci to understand ES6

**Review by gik-hub — APPROVED** [review]
Great.

**Review by Jordybastien — APPROVED** [review]
LGTM

**Review by Paulvitalis200 — APPROVED** [review]
Good work on this chore.

**Pull request merged**